src/AccessibilityInsights.SharedUx/Settings/ConfigurationModel.cs

```diff
@@ -631,7 +631,7 @@ public static ConfigurationModel GetDefaultConfigurationModel(FixedConfigSetting
                 FontSize = FontSize.Standard,
                 HighlighterMode = HighlighterMode.HighlighterBeakerTooltip,
                 ShowAncestry = true,
-                EnableTelemetry = true,
+                EnableTelemetry = false,
```

Comment:
This isn't strictly necessary but better reflects what the default value should be